### ISSUE TITLE
Fractional delay input range

### ISSUE THREAD

**Issue by ebezzam** [LCAV/pyroomacoustics#56]
https://github.com/LCAV/pyroomacoustics/blob/0d6025f67ca60ca78e97c867cf4b794a67211794/pyroomacoustics/utilities.py#L369

Values within `[-1,1]` are also accepted. Can be edited in the next PR.

**Comment by fakufaku** [comment]
Yes, I'll do a quick commit in the master branch if that's ok. Would the following be ok:
```
The delay in fraction of sample. Typically between -1 and 1.
```
?

**Comment by ebezzam** [comment]
Yeah thats perfect 👍

**Comment by fakufaku** [comment]
Fixed by 5e565e898
